### PR TITLE
ArgumentRemoverRectorTest: replace symfony\yaml reference with dummy

### PR DESCRIPTION
**Pull request by janedbal**
- https://github.com/rectorphp/rector-src/pull/5777#discussion_r1541929575

### CODE REVIEW

File: rules-tests/Removing/Rector/ClassMethod/ArgumentRemoverRector/config/configured_rule.php [hunk 1]
```diff
@@ -7,16 +7,17 @@
...
     $rectorConfig
         ->ruleWithConfiguration(ArgumentRemoverRector::class, [
-            new ArgumentRemover(Persister::class, 'getSelectJoinColumnSQL', 4, null), new ArgumentRemover(
+            new ArgumentRemover(Persister::class, 'getSelectJoinColumnSQL', 4, null),
+            new ArgumentRemover(
                 Yaml::class,
                 'parse',
                 1,
-                ['Symfony\Component\Yaml\Yaml::PARSE_KEYS_AS_STRINGS', 'hey', 55, 5.5]
+                [Yaml::REMOVED_CONSTANT, 'hey', 55, 5.5]
```
Comment (samsonasik) [inline rules-tests/Removing/Rector/ClassMethod/ArgumentRemoverRector/config/configured_rule.php]: should keep using string `Symfony\Component\Yaml\Yaml::PARSE_KEYS_AS_STRINGS` reference, `Symfony\Component\Yaml\Yaml` no need to be exists to make transformation happen.

Comment (janedbal) [reply rules-tests/Removing/Rector/ClassMethod/ArgumentRemoverRector/config/configured_rule.php]: Adjusted.

**Comment by samsonasik** [comment]
You need to rebase to make CI green

**Comment by TomasVotruba** [comment]
Thank you :tada:

**Pull request merged**